.github/DISCUSSION_TEMPLATE/Help.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Discussion
+about: Describe a problem or seek help from the community
+
+---
+
+## Description
+<!-- A clear and concise description of the issue, and why you consider it to be a bug. -->
```

Comment:
_Help_ ticket it's not always a bug, sometimes it represents some problem that developer couldn't solve and asks for additional information, like Info Requests that we have. Bugs should be create in _Issues_ tab, it's what we have now